### ISSUE TITLE
Allow running the manual tests

### ISSUE THREAD

**Issue by pomek** [ckeditor/ckeditor5-dev-tests#27]
Karma web-server allows opening additional HTML files. It is helpful for running the manual tests.

We need to create some mechanism which will compile all manual tests. Compilation means - for each manual test:

1. [Concat the basic HTML template with a template for a test](https://github.com/benderjs/dom-combiner).
2. Pass to the Webpack script with an initial behavior for the test.

The basic HTML template most probably is a simple DOM:

```html
<!doctype html>
<html lang="en">
<head>
	<meta charset="UTF-8">
</head>
<body>
</body>
</html>
```

We also need to have a list which will contain available manual tests.

I don't know how to connect the automatic tests with the manual tests. I mean - do we want to create a new gulp task or try to integrate existing one.

If we would like to integrate with existing task, we should consider how will look the new CLI API.

**Comment by Reinmar** [comment]
> I don't know how to connect the automatic tests with the manual tests. I mean - do we want to create a new gulp task or try to integrate existing one.

I think it can be `gulp test:manual` for simplicty. It will be easier to extend it (with e.g. `--watch` option) if it's not part of the base task.

Actually... you've been thinking about using Karma for this, but do we need Karma? We need a simple HTTP server and that's it.

**Comment by pomek** [comment]
Yep, we don't need Karma. We can use [`http-server`](https://www.npmjs.com/package/http-server) instead of. It allows to running automatic and manual tests in parallel.

**Comment by pomek** [comment]
The absolute paths in manual tests have to be changed (the same like in automatic tests).

```diff
- import ClassicEditor from '/ckeditor5/editor-classic/classic.js';
+ import ClassicEditor from 'ckeditor5/editor-classic/classic.js';
```
